app/templates/@layout.latte

```diff
@@ -63,6 +63,7 @@
 		{block main}
 		<div class="main" lang="en">
 			<div class="content has-sidebar">
+				<div class="flash error">See our new modern addons portal <a href="http://bit.ly/cmtte">componette.com</a>. Completely automatic.</div>
```

Comment:
https://bit.ly/cmtte works too, but :-1: for using external service for simple redirect.